packages/roosterjs-content-model/lib/editor/plugins/PastePlugin/WordDesktop/processWordLists.ts

```diff
@@ -12,6 +13,11 @@ const MSO_LIST = 'mso-list';
 const MSO_LIST_IGNORE = 'ignore';
 const LOOKUP_DEPTH = 5;
 
+interface WordDesktopListFormat extends DomToModelListFormat {
+    wordLevel: number | '';
+    wordKnownLevels: Map<string, ContentModelListItemLevelFormat[]>;
```

Comment:
Let's make these 2 properties optional, so that in other code compiler can make sure we do null check